### ISSUE TITLE
Colours fail WCAG 2.0 1.4.3

### ISSUE THREAD

**Issue by MichielBijl** [w3c/wptreport#21]
Most colours used in [the report](https://w3c.github.io/test-results/wai-aria/all.html) fail minimum contrast.
